docs/content/concepts/configuration/config-schema-legacy.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Run Configuration (Legacy) | Dagster
+description: Job run configuration allows providing parameters to jobs at the time they're executed.
+---
+
+# Run Configuration (Legacy)
+
+<Note>
+  This guide covers using the legacy Dagster config system. For docs on the new
+  Pythonic config system introduced in Dagster 1.3, see the [updated
+  configuration guide](/concepts/configuration/config-schema).
```

Comment:
```suggestion
  Pythonic config system introduced in Dagster 1.3, see the <a href="/concepts/configuration/config-schema">updated
  configuration guide</a>.
```

An unfortunate thing about the `Note` component is that we can't use Markdown for formatting. HTML only -.-